packages/integrations/sitemap/src/index.ts

```diff
@@ -32,7 +32,7 @@ export type SitemapOptions =
 			entryLimit?: number;
 
 			// sitemap specific
-			changefreq?: ChangeFreq;
+			changefreq?: EnumChangefreq;
```

Comment:
this can be technically be replaced with
```suggestion
			changefreq?: typeof ChangeFreq[keyof typeof ChangeFreq];
```
if we don't want to reuse `EnumChangefreq` but `EnumChangefreq` is way cleaner and readable